docs/src/pages/en/themes/handling-variables.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Handling variables
+description: Learn how to handle variables in Media Chrome Themes
+layout: ../../../layouts/MainLayout.astro
+---
+
+Custom variables that the theme author defines can be provided by users of your
+theme through attributes and then used inside your theme with double curly brackets,
+for example `{{ username }}`
+
+If you would like to provide a fallback for a variable that might be empty,  
+this can be done like so `{{username ?? 'Unknown username'}}`.
+
+```html
+<template id="vars-theme">
+  <media-controller>
+    <slot name="media" slot="media"></slot>
+    <media-text-display slot="top-chrome">
+      {{username ?? 'Unknown user'}}
+    </media-text-display>
+  </media-controller>
+</template>
+
+<media-theme template="vars-theme" username="bobbytables">
+  <video
+    slot="media"
+    src="https://stream.mux.com/A3VXy02VoUinw01pwyomEO3bHnG4P32xzV7u1j1FSzjNg/high.mp4"
+  ></video>
+</media-theme>
+```
+
+<br>
+
+<template id="vars-theme">
+  <media-controller>
+    <slot name="media" slot="media"></slot>
+    <media-text-display slot="top-chrome">
+      {{username ?? 'Unknown username'}}
+    </media-text-display>
+  </media-controller>
+</template>
+
+<media-theme template="vars-theme" username="bobbytables">
+  <video
+    slot="media"
+    src="https://stream.mux.com/A3VXy02VoUinw01pwyomEO3bHnG4P32xzV7u1j1FSzjNg/high.mp4"
+  ></video>
+</media-theme>
+
+[![Edit Media Chrome Vars Theme](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/media-chrome-vars-theme-nejd49?fontsize=14&hidenavigation=1&theme=dark)
+
+In the example above the `username` attribute is provided to the `<media-theme>` 
+element and this is then rendered in the template.
+
+## Avoid native attribute names
+
+Avoid using native HTML attributes for variables. This includes things like:
+
+- `title`. The native `title` attribute will add a tooltip on mouse hover. You can 
+use something like `videotitle` instead.
+- `style`. The native `style` attribute is used for CSS
+
+Also avoid attribute names that conflict with the **Special variables** listed below.
```

Comment:
If both are provided, the special variables take precedence? If so, I think it's worth being explicit here (or in the section below).

Reply:
theme attrs override the special attrs currently, added a line on that